DX-1299 Øyvind's Payer Aware Payment Menu Changes (#767)
SwedbankPay/developer.swedbankpay.com: _includes/payment-order-other-features.md

their payment details, so even with `generatePaymentToken` set to `true`, it is
still up to the payer if they want the details stored or not.

Remember that you will have the responsibility to enforce GDPR requirements and
let the payer remove active payment tokens when they want. It is up to you how
to implement this functionality on your side but Swedbank Pay has the API you
need to ensure that cleaning up old data is easy. It is possible to query for
all active payment tokens registered on a specific `payerReference`. This query
will also list the operations used for deleting tokens. You can choose to remove
all tokens for that payer or only a subset of all tokens connected to that

> `payerReference`.

Reviewer: Convert a part of this paragraph into an alert and then move the alert below further up so they aren't right next to each other.

Author: Which part do you want me to convert?

Reviewer: The first sentence, perhaps?

{% include alert.html type="informative" icon="info" body="Please note that not
all payment instruments provided by Swedbank Pay support Payer Awareness today."

> %}

Reviewer: Move this alert further up.

their payment details, so even with `generatePaymentToken` set to `true`, it is
still up to the payer if they want the details stored or not.

> {% include alert.html type="informative" icon="info" body="Remember that the

Reviewer: Let's add a `header` to this alert as well? Saying something like "GDPR Responsibility".

responsibility to enforce the GDPR requirements lies on you as a merchant."

> %}

Reviewer: Don't replace the content of this alert. We should have two alerts for this section:

1. Explaining that Payer Awareness is not available in all payment instruments.
2. Detailing the GDPR responsibility.

These two alerts need to be placed apart so they aren't displayed right after each other, as that would look weird. So that's why I suggested to move the existing alert up and create a new alert for the new GDPR section.

Reviewer: Ah, that was weird. I now see the first alert as well. The placement is perfect, please just ignore me. 😅

Payment Menu is versatile and can be configured in such a way that it functions
like a single payment instrument. In such configuration, it is easy to Bring
Your Own Payment Menu, i.e. building a customized payment menu in our own user

> interface. For InApp integrations, the payers' shopping experience will be

Reviewer: ```suggestion
interface. For in-app integrations, the payers' shopping experience will be
```

better if you choose the BYO Payment Menu option. For web integrations, we
highly recommend the Swedbank Pay Payment Menu.

> The `payerReference` can be used to list all active tokens here as well.This

Reviewer: ```suggestion
The `payerReference` can be used to list all active tokens here as well. This
```

> will also list the payment instruments listed in the `availableInstruments`, as

Reviewer: ```suggestion
will also list the payment instruments enumerated in the `availableInstruments`, as
```

instrument-mode is mandatory when you are using BYO Payment Menu.

The alternative to the `payerReference` option is to store `paymentToken`s found
in the `paymentOrder`s after the payment is completed. This requires that the

> merchant makes sure this list is up to date at their end.

Reviewer: ```suggestion
merchant makes sure the list of `paymentToken`s is up to date at their end.
```

            "productCategory": "A123",
            "orderReference": "or-123456",
            "subsite": "MySubsite"
        },
        "payer": {  

>             "payerReference": "AB1234",

Reviewer: 👍 I do think there are more of these `payerReference` fields placed wrong. Can you please do a search and replace on the entire documentation?

Author: Roger that, Houston!

> {% if include.documentation_section == "payment-menu" %}

Reviewer: Should this `if` be deleted alongside its corresponding `endif`? The build is currently failing because they are not paired properly.

Author: Ah, I was looking for that one. Will remove it.

> {% if include.documentation_section == "payment-menu" %}

Author: ```suggestion
```
